### ISSUE TITLE
Filter: Neighborhoods/Community Org/Zone Pills: Unable to clear the pills using screen readers

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#653]
## Description

Clearing the pills is impossible using NVDA & Voice Over. JAWS is more forgiving.

NVDA + FF/Edge: Using tab key to navigate to the X button, selecting it activates the listbox pop up instead
VO + Safari: Screen reader focus could not navigate to the pills at all; no feedback given when tabbing to it.

## Recommendation
- Would be better to not put the pill on top of the listbox popup so that there's a clear separation between the 2. It will also make the UI less confusing even for sighted users.
